app/components/citation_component.rb

```diff
@@ -0,0 +1,28 @@
+# typed: false
+# frozen_string_literal: true
+
+# A button that opens a modal containing the citation for a deposit
+class CitationComponent < ApplicationComponent
+  def initialize(work_version:)
+    @work_version = work_version
+  end
+
+  def call
+    tag.button data: {
+      controller: 'show-citation',
+      action: 'show-citation#setContent',
+      show_citation_citation_value: work_version.citation,
+      show_citation_header_value: work_version.title,
+      show_citation_target_value: '#citationModal',
+      bs_toggle: 'modal',
+      bs_target: '#citationModal'
+    }, class: 'btn' do
+      # It's a SafeBuffer, not a String
+      # rubocop:disable Style/StringConcatenation
+      tag.span(class: 'fas fa-quote-left') + ' Cite'
+      # rubocop:enable Style/StringConcatenation
```

Comment:
If you're open to doing this in one line:

```suggestion
      tag.span(class: 'fas fa-quote-left') + ' Cite' # rubocop:disable Style/StringConcatenation
```